Modules/_cursesmodule.c

```diff
@@ -3408,7 +3564,7 @@ _curses_initscr_impl(PyObject *module)
     WINDOW *win;
 
     if (curses_initscr_called) {
-        wrefresh(stdscr);
+        (void)wrefresh(stdscr); // TODO(picnixz): should we report an error?
```

Comment:
Please open issues for TODOs.

Reply:
Sorry, I'll do so when I'm back (in 10 days)

Reply:
See https://github.com/python/cpython/issues/133579.